utils.js

```diff
@@ -101,6 +108,23 @@ export function convertNumberToBase(str, fromBase, toBase, targetLen = 0) {
     return res;
 }
 
+const UID_LENGTH = 32;
+const UID_DASH_INDICES = [0, 8, 12, 16, 20, 32];
+
+/**
+ * UID in this instance being any string that was formatted as ( xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx )
+ * before it was converted to Base-64
+ * @param {string} UID 
+ * @returns 
+ */
+export function revertUID(UID) {
```

Comment:
noice. Let's add unit tests in another PR so we can watch the Coverage number go up